client/src/configuration.rs

```diff
@@ -1039,7 +1039,7 @@ pub fn parse_hex_string<F: FromStr>(hex_str: &str) -> Result<F, F::Err> {
 pub fn parse_config_address_string(
     addr: &str, network: &Network,
 ) -> Result<Address, String> {
-    let base32_err = match serde_json::from_str::<RpcAddress>(addr) {
```

Comment:
We can simply add "" around the addr?

Reply:
I've thought about this solution, but it has a side effect to make the error report less intuitive if the address format is incorrect...